library/math/numbersLine/src/components/MoveableElement.tsx

```diff
@@ -21,10 +26,11 @@ const MoveableElement = ({ targetRef }: IProps) => {
       renderDirections={["w", "e"]}
       onScale={(e) => {
         e.target.style.transform = e.drag.transform;
+        updateTransform(e);
+        console.log(e);
```

Comment:
remove log.

Reply:
and e.target.style.transform = e.drag.transform; put on function updatetransform